app/views/sponsor/contributions/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Index Page for Sponsors Contributions</h1>
+<p>This is the page all contributions will be listed</p>
+<% @contributions.each do |contribution|%>
```

Comment:
@contributions should be one for the projects created by sponsor (current_user).
Hint:
current_user.projects => array of project
loop through projects to get contributions and return an array of only pending contributions.